platform/ios/src/MGLMapView.mm

```diff
@@ -5280,9 +5280,9 @@ - (UIEdgeInsets)edgePaddingForFollowing
                                                    correctPoint.x - CGRectGetMidX(bounds),
                                                    correctPoint.y - CGRectGetMidY(bounds));
     return UIEdgeInsetsMake(CGRectGetMinY(boundsAroundCorrectPoint) - CGRectGetMinY(bounds),
-                            self.contentInset.left,
+                            CGRectGetMaxX(boundsAroundCorrectPoint) - CGRectGetMaxX(bounds),
                             CGRectGetMaxY(bounds) - CGRectGetMaxY(boundsAroundCorrectPoint),
-                            self.contentInset.right);
+                            CGRectGetMaxX(bounds) - CGRectGetMaxX(boundsAroundCorrectPoint));
```

Comment:
This change fixes an issue where the user location annotation was positioned incorrectly when the map view had a left or right content inset. It’s worth mentioning in the changelog as well.